README.md

```diff
@@ -6,9 +6,23 @@
 [![Nordic ClangFormat](https://img.shields.io/static/v1?label=Nordic&message=ClangFormat&labelColor=00A9CE&color=337ab7)](https://github.com/nrfconnect/sdk-nrf/blob/master/.clang-format)
 [![Zephyr compliance](https://img.shields.io/static/v1?label=Zephry&message=compliance&labelColor=4e109e&color=337ab7)](https://docs.zephyrproject.org/latest/contribute/index.html#coding-style)
 
+It determines NAT timeouts in a cellular networks by sending messages to the [NAT-TestServer](https://github.com/NordicSemiconductor/NAT-TestServer) and waiting for the reply.
+
 This application is built with [sdk-nrf](https://github.com/nrfconnect/sdk-nrf).
 
-Used to test NAT timeouts in different networks, sends messages to [NAT-TestServer](https://github.com/NordicSemiconductor/NAT-TestServer).
+## Getting started
+
+Please help us learn more about NAT timeouts in cellular networks:
```

Comment:
Support this project by giving more insight into the NAT timeouts in cellular networks.
```suggestion
Support this project by giving more insight into the NAT timeouts in cellular networks.
The following steps describe the testing process:
```
The following steps describe the testing process: